src/openjd/model/_step_param_space_iter.py

```diff
@@ -82,48 +106,84 @@ def __init__(self, *, space: Optional[StepParameterSpace]):
                 combination = space.combination
             self._parameters = dict(space.taskParameterDefinitions)
 
+            # Determine whether and what kind of chunking to do
+            for name, param in self._parameters.items():
+                if param.type == TaskParameterType_2023_09.CHUNK_INT:
+                    if param.chunks is None:
+                        raise ValueError(
+                            f"CHUNK[INT] parameter '{name}' must have a chunks definition."
+                        )
+                    self._chunks_adaptive = (
+                        param.chunks.targetRuntimeSeconds is not None
+                        and int(param.chunks.targetRuntimeSeconds) > 0
+                    )
+                    self._chunks_default_task_count = int(param.chunks.defaultTaskCount)
+                    break
```

Comment:
We break out of the loop after finding the first parameter of type `CHUNK[INT]` here which led me to wonder if we only allow a single parameter of type `CHUNK[INT]` in a step parameter space.

After reviewing the RFC and OpenJobDescription specification, it was unclear that only one such parameter is allowed. Discussed with @mwiebe  offline and we agree that we need to amend the RFC, specification, and the model validation to model this assumption.

The plan is to add the validation for a single `CHUNK[INT]` parameter for a step parameter space in this PR and then back-fill the specification and RFC changes.

Reply:
I've added validation and a test for ensuring a single `CHUNK[INT]` parameter.

Reply:
We've merged a spec clarification here: https://github.com/OpenJobDescription/openjd-specifications/pull/73